### PR TITLE
event funding: use the new GH issue workflow

### PR DESCRIPTION
**Pull request by zimbatm**
Also part of https://github.com/NixOS/foundation/issues/73

The form has the same fields as the old one (with minor variations), but it will be easier for us to track the new requests

### CODE REVIEW

**Review by garbas — APPROVED** [review]

**Comment by garbas** [comment]
The build is failing and I need to figure out why. Maybe it was intermittent.

**Comment by zimbatm** [comment]
I think it's the `&` that needs to be escaped. I'll take a look once I get Domen's feedback on the form content.

File: community/event-funding.tt [hunk 1]
```diff
@@ -69,7 +69,7 @@ For instance, you can apply if you're planning to organize:
   </p>
     
   <div class="community-event-funding-button">
-    <a href="https://commotion.page/view/event-funding">Apply</a>
+    <a href="https://github.com/NixOS/foundation/issues/new?labels=funding-request%2Ctriage&template=funding_form.yml&title=%5BFunding%5D%3A+%24event+name+%24date">Apply</a>
```
Comment (garbas) [inline community/event-funding.tt:72]:
```suggestion
    <a href="https://github.com/NixOS/foundation/issues/new?labels=funding-request%2Ctriage&amp;template=funding_form.yml&amp;title=%5BFunding%5D%3A+%24event+name+%24date">Apply</a>
```
Maybe this is a fix

**Comment by zimbatm** [comment]
Alright, I guess that Domen is budy. It's probably good enough to merge.

**Pull request merged**